thirdeye/thirdeye-pinot/src/test/java/org/apache/pinot/thirdeye/integration/AnomalyApplicationEndToEndTest.java

```diff
@@ -1,435 +0,0 @@
-/**
```

Comment:
Why delete this test? Looks like it's still relevant.

Reply:
This test is still relying on the legacy classes like DetectionJobScheduler and AlertJobSchedulerV2. It needs to be rewritten.